vmm/src/config.rs

```diff
@@ -452,6 +454,11 @@ pub struct CpuAffinity {
     pub host_cpus: Vec<u8>,
 }
 
+#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
+pub struct CpuFeatures {
+    pub amx: bool,
```

Comment:
Did you need to make this conditional on `#[cfg(all(feature = "amx", target_arch = "x86_64"))] `?